src/views/product-tutorials-view/components/sitemap/sitemap.module.css

```diff
@@ -1,4 +1,98 @@
-.root {
-  border: 1px solid magenta;
+.headingBar {
+  display: flex;
+}
+
+.heading {
+  composes: hds-typography-display-300 from global;
+  font-weight: var(--token-typography-font-weight-bold);
+  margin: 0 auto 0 0;
+}
+
+.collectionListRoot {
+  margin: 32px 0 0 0;
+  list-style: none;
+  padding: 0;
+}
+
+.collectionListItem {
+  &:not(:first-child) {
+    margin-top: 32px;
+  }
+}
+
+/**
+ * .collectionLinkSizer makes it so .collectionLink is NOT full width.
+ * An alternative might be display: inline-flex on .collectionLink,
+ * chose this instead cause it makes vertical spacing easier to reason about.
+ */
+.collectionLinkSizer {
+  display: flex;
+  align-items: flex-start;
+}
+
+.collectionLink {
+  composes: hds-typography-body-200 from global;
+  align-items: center;
+  color: var(--token-color-foreground-primary);
+  display: flex;
+  font-weight: var(--token-typography-font-weight-medium);
+  text-decoration: underline;
+}
+
+.collectionLinkIcon {
+  display: block;
+  flex-shrink: 0;
+  margin-right: 8px;
+}
+
+.tutorialListRoot {
+  /* note: top & bottom margin adjusted for use of 4px padding in list items */
+  margin: 12px 0 -4px 0;
+  list-style: none;
+  padding: 0;
+
+  &.hasMultipleItems {
```

Comment:
Some collections only have one tutorials, and this one tutorial may have a very long name. In these cases, it feels odd to force the tutorial into a half-width column, potentially causing text to wrap.